### ISSUE TITLE
Non-matching definitions between .owl file and FOIS 2023 paper. 

### ISSUE THREAD

**Issue by giacomodecolle** [Occupation-Ontology/OccO#12]
Hello,

the current definitions of most important terms in OccO are not corresponding to the definitions provided in the FOIS 2023 paper. By "most important terms", I refer to occupation holder, occupation disposition, occupation role, occupation activity, skill, ability.

Expected behavior would be to align the current definitions in the owl file to the definitions provided in the FOIS 2023 paper.

**Comment by zhengj2007** [comment]
@giacomodecolle Could you please add the definitions of the terms that you find not correctly represented in the ontology in the issue? So, we can update them in the OccO ontology. Thanks!

**Comment by zhengj2007** [comment]
> occupation role: A role borne by a human that, if realized, is realized when the bearer provides labor or services in exchange for compensation as specified by some deontic declaration.

Is it same as OMRSE: [job role](http://purl.obolibrary.org/obo/OMRSE_00000077)? 
A role in human social processes that, if realized, is realized when the bearer provides labor or services in exchange for a wage or salary as specified by some deontic declaration.

If so, OccO needs to import OMRSE job role.

**Comment by giacomodecolle** [comment]
Yes, I think that as it stands the terms might be equivalent. I think that OMRSE is currently working on an update so we might have to wait until their update is ready to import the term. Given that we do have an OccO occupation role, we could update the definition in the meantime in order to reflect the paper and then eventually substitute it with the OMRSE term when the time comes. When you make a decision about these changes, would you please let me know so that I can update my fork of OccO and then open a pull request with the work I have done on skills definitions?

**Comment by zhengj2007** [comment]
I'd like to discuss the issue on the OccO meeting before make any changes.
